tests/integration/bond_test.py

```diff
@@ -353,6 +353,73 @@ def test_changing_slave_order_keeps_mac_of_existing_bond(bond99_with_2_slaves):
     )
 
 
+def test_adding_a_slave_keeps_mac_of_existing_bond(bond99_with_eth2, eth1_up):
```

Comment:
please mark this and the other working tests as tier1

Reply:
Done, thanks!